# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_wget.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/progress.c~	Fri May 24 14:58:17 2002
+++ ./src/progress.c	Sat Mar 15 20:48:35 2003
@@ -690,7 +690,7 @@
   else if (bp->total_length > 0)
     APPEND_LITERAL ("    ETA --:--");
 
-  assert (p - bp->buffer <= bp->width);
+  assert ((p - bp->buffer) <= bp->width);
 
   while (p < bp->buffer + bp->width)
     *p++ = ' ';
--- ./src/url.c~	2003-11-14 12:25:27.000000000 +0100
+++ ./src/url.c	2003-11-14 12:28:43.000000000 +0100
@@ -204,7 +204,7 @@
       else
 	*p2++ = *p1++;
     }
-  assert (p2 - newstr == newlen);
+  assert ((p2 - newstr) == newlen);
   *p2 = '\0';
 
   return newstr;
@@ -398,7 +398,7 @@
 	}
     }
   *p2 = '\0';
-  assert (p2 - newstr == newlen);
+  assert ((p2 - newstr) == newlen);
   return newstr;
 }
 
@@ -1481,7 +1481,7 @@
 	      *q++ = XNUM_TO_DIGIT (ch & 0xf);
 	    }
 	}
-      assert (q - TAIL (dest) == outlen);
+      assert ((q - TAIL (dest)) == outlen);
     }
   TAIL_INCR (dest, outlen);
 }
@@ -2018,7 +2018,7 @@
   p += fplen;
   *p++ = '\0';
 
-  assert (p - result == size);
+  assert ((p - result) == size);
 
   if (quoted_user && quoted_user != url->user)
     xfree (quoted_user);
--- ./src/html-parse.c~	2003-11-14 12:37:26.000000000 +0100
+++ ./src/html-parse.c	2003-11-14 12:37:59.000000000 +0100
@@ -379,7 +379,7 @@
 	}
       /* Verify that we haven't exceeded the original size.  (It
 	 shouldn't happen, hence the assert.)  */
-      assert (to - (pool->contents + pool->tail) <= end - beg);
+      assert ((to - (pool->contents + pool->tail)) <= (end - beg));
 
       /* Make POOL's tail point to the position following the string
 	 we've written.  */
--- ./src/convert.c~	2003-11-14 12:36:33.000000000 +0100
+++ ./src/convert.c	2003-11-14 12:37:06.000000000 +0100
@@ -600,7 +600,7 @@
 	      *to++ = 'F';
 	    }
 	}
-      assert (to - newname == fsqlen);
+      assert ((to - newname) == fsqlen);
       *to = '\0';
 
       file_sans_qmark = newname;
